frontend/test/metabase/scenarios/question/timelines.cy.spec.js

```diff
@@ -65,7 +65,9 @@ describe("scenarios > collections > timelines", () => {
 
       cy.findByLabelText("calendar icon").click();
       cy.findByText("Releases");
```

Comment:
nit: Move `cy.findByText("Releases");` into `sidebar().within` because we expecting this in the sidebar